commands/help.go

```diff
@@ -1,24 +1,28 @@
 package commands
 
 import (
+	"context"
+
 	"github.com/spf13/cobra"
-	"golang.org/x/exp/slices"
 )
 
 func init() {
-	RootCmd.SetHelpFunc(HelpCmd.Run)
-	RootCmd.AddCommand(HelpCmd)
+	RootCmd.SetHelpCommand(HelpCmd)
 }
 
 var HelpCmd = &cobra.Command{
 	Use:                "help",
 	FParseErrWhitelist: cobra.FParseErrWhitelist{UnknownFlags: true},
 	Run: func(cmd *cobra.Command, args []string) {
-		if len(args) == 0 {
-			args = []string{"list"}
-		} else if !slices.Contains(args, "--help") && !slices.Contains(args, "-h") {
-			args = append([]string{"help"}, args...)
+		cmd.SetContext(context.Background())
```

Comment:
You should not change the context here - is there a need for that?

Reply:
no, you are right. deleted it